sql/pre_install/tables.sql

```diff
@@ -495,6 +495,7 @@ CREATE TABLE _timescaledb_catalog.compression_chunk_size (
   compressed_index_size bigint NOT NULL,
   numrows_pre_compression bigint,
   numrows_post_compression bigint,
+  numrows_frozen bigint,
```

Comment:
This doesn't reflect the actual frozen status of rows after e.g. `VACUUM FREEZE` runs, does it? If it's only for testing, might be better to drop it altogether not to confuse the users. For testing we can use a debug GUC that checks that the tuples are frozen on insert.

Reply:
The catalog change is performed to have the data available for the telemetry. But you are right `numrows_frozen` might be a bit confusing since it is not related to the tuples that are frozen by `VACUUM FREEZE`. 

What do you think about the name `numrows_frozen_immediately` to make it more clear?

Reply:
Discussed on slack and changed to `numrows_frozen_immediately`